module-types/Extreme Networks/EC72051AF-E6.yaml

```diff
@@ -0,0 +1,11 @@
+---
+manufacturer: Extreme Networks
+model: EC72051AF-E6
```

Comment:
I am having a hard time finding `EC72051AF-E6`, but I have found `EC7205A1F-E6`

Is the proper model name `EC7205A1F-E6`?

Reply:
This is indeed a typo. Made the change.